.github/workflows/tests.yml

```
@@ -33,8 +34,16 @@ jobs:
         with:
           node-version: ${{matrix.node_version}}
 
+      # See https://github.com/npm/cli/issues/2610
+      - name: Hack for Node 14
+        run: |
+          if [[ "${{matrix.node_version}}" = "14" ]]; then
+            sed -i 's/git+ssh/git+https/g' package-lock.json
+          fi
+          exit 0
```

GitHub Actions allows to use if conditions.
This is a bit cleaner and skips the step altogether for other versions.

```suggestion
      # See https://github.com/npm/cli/issues/2610
      - name: Workaround for Node 14
        if: ${{matrix.node_version}} = "14"
        run: sed -i 's/git+ssh/git+https/g' package-lock.json
```

---

Fixed my error in the if condition. Works now as intended.